docs(examples): GDPR Cookie Consent (#1428)
remix-run/remix: examples/gdpr-cookie-consent/README.md

# GDPR Cookie Consent

Create a simple GDPR consent form.
Till the user doesn't click on the `Accept` button, she will see a banner prompting to accept cookies.

## Preview

Open this example on [CodeSandbox](https://codesandbox.com):

[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/gdpr-cookie-consent)

## Example

> Users will be presented with a GDPR consent form on every page ([app/routes/index.tsx](app/routes/index.tsx)) till they submit the accept button.

Reviewer: If the form is in the index and not the root, then it’s not going to be on every page

# GDPR Cookie Consent

Create a simple GDPR consent form.
Till the user doesn't click on the `Accept` button, she will see a banner prompting to accept cookies.

## Preview

Open this example on [CodeSandbox](https://codesandbox.com):

> <!-- TODO: update this link to the path for your example: -->
> 

Reviewer 2: ```suggestion
```

> [![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/remix-gdpr-example-e7znt?file=/app/routes/index.tsx)

Reviewer 2: ```suggestion
[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/gdpr-cookie-consent)
```

# GDPR Cookie Consent

Create a simple GDPR consent form.
Till the user doesn't click on the `Accept` button, she will see a banner prompting to accept cookies.

## Preview

Open this example on [CodeSandbox](https://codesandbox.com):

[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/gdpr-cookie-consent)

## Example

Users will be presented with a GDPR consent form on every page [app/root.tsx](app/root.tsx) till they submit the accept button.

> Once they submit the consent form, a dummy tracking [script](public/dummy-analytics-script.js) at the [app/root.tsx](app/root.tsx)  will start tracking the user`s data (open the browser console too see the dummy tracking message).

Reviewer 3: ```suggestion
Once they submit the consent form, a dummy tracking [script](public/dummy-analytics-script.js) at the [app/root.tsx](app/root.tsx) will start tracking the user's data (open the browser console too see the dummy tracking message).
```
